core/modules/utils/dom/dragndrop.js

```diff
@@ -220,4 +220,10 @@ exports.dragEventContainsFiles = function(event) {
 	return false;
 };
 
+exports.dragEventContainsFiles = function(event) {
+	return (dragEventContainsType(event,"Files") && !$tw.utils.dragEventContainsType(event,"text/plain"));
```

Comment:
I was confused by the two different ways of addressing the dragEventContainsType function. Could it be like this:

```
return (dragEventContainsType(event,"Files") && !dragEventContainsType(event,"text/plain"));
```

Reply:
@Jermolene you are absolutely right, that is an artefact from a late refactoring. Will resolve now.